README.md

```diff
@@ -1,2 +1,346 @@
-# lean-html5
-Learn how to use HTML5 (from scratch) to build websites/web applications!
+# Learn HTML5
+Learn how to use HTML5 (from scratch) to build websites/web applications! No previous knowledge needed.
+
+## Some history
```

Comment:
@karinakozarova We like to [start our tutorials with why](https://www.ted.com/talks/simon_sinek_how_great_leaders_inspire_action), but this isn't a blocker and I love that you have a little history in here. In a future iteration we can add it in 👍